concourse/templates/default.yaml

```diff
@@ -79,8 +80,8 @@ scan_container_images_step = concourse.steps.step_def('scan_container_images')
 <%namespace file="/resources/meta.mako" import="*"/>
 
 inherit:
-${image_registry_defaults(registry_name='gcr', registry_cfg=gcr)}
-${task_image_resource(registry_name='gcr')}
+${image_registry_defaults(registry_name=default_container_registry_name, registry_cfg=default_container_registry)}
```

Comment:
interesting.. what does the registry_name mean, actually? I completely forgot about this piece of coding